src/crypto/x509/pkcs8.go

```diff
@@ -74,6 +76,20 @@ func ParsePKCS8PrivateKey(der []byte) (key interface{}, err error) {
 		}
 		return ed25519.NewKeyFromSeed(curvePrivateKey), nil
 
+	case privKey.Algo.Algorithm.Equal(oidPublicKeyEdDilithium3):
+		if l := len(privKey.Algo.Parameters.FullBytes); l != 0 {
+			return nil, errors.New("x509: invalid Ed25519-Dilithium3 private key parameters")
+		}
+		var packedSk []byte
```

Comment:
Let's change the 'packedSk' name to 'curvePrivateKey' ;)

Reply:
But it does not necessarily have anything to do with a curve, does it?

Reply:
(I'm reading your comment as if you've made it on the latest version with he generic code.)

Reply:
No, it is more around following the names that are already there.